src/components/pages/home/index.tsx

```diff
@@ -95,193 +105,223 @@ const Home: FunctionComponent<any> = ({homePageData}) => {
           />
         )}
       </div> */}
-      <div className="lg:space-y-6 space-y-4">
-        <Jumbotron resource={jumbotron} />
-        <section className="">
+      <div className="mt-12">
+        <WhatsNew resource={featureWhatsNew} />
```

Comment:
What's new now replaces the Jumbotron (uses it inside the component). This can be updated with sanity